smart-client-jersey/src/main/java/com/emc/rest/smart/jersey/SmartFilter.java

```diff
@@ -75,9 +76,11 @@ public ClientResponse handle(ClientRequest request) throws ClientHandlerExceptio
             return response;
         } catch (RuntimeException e) {
 
-            // capture requests stats (error)
-            host.callComplete(true);
-            host.connectionClosed();
+            if (e instanceof SmartClientException && !((SmartClientException)e).getErrorType().equals(SmartClientException.ErrorType.Client)) {
+                // capture requests stats (error)
```

Comment:
so, we **always** have to call `callComplete()` - we are just changing whether the call was an error (`true`) or not (`false`) - if it was a _client_ error, that will be `false`; otherwise, it should be `true`.  So really, you are just changing the parameter sent to `callComplete()`